src/portals/hope/features/search/members/MemberSearch.tsx

```diff
@@ -64,6 +64,24 @@ export const MemberSearch: React.FC<{ query: string | null }> = ({
       })
   }, [defaultQuery])
 
+  const luckySearchHandler = async () => {
+    if (!query) {
+      return
+    }
+
+    await search({
+      type: hasSearchCarMembersPermission ? 'CAR_MEMBERS' : 'MEMBERS',
+    })
+
+    if (hits.length) {
+      const luckyHit = hits[0]
+
+      const link = `/members/${luckyHit.hit.memberId}/contracts`
+
+      history.push(link)
```

Comment:
```suggestion
      history.push(`/members/${hits[0].hit.memberId}/contracts`)
```
Think this code is clear enough 🧐